config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "customers/:id", to: "customers#show"
+  post "customers/:customer_id/items", to: "customer_items#create"
```

Comment:
I like this choice to use this nested route and to sent it to the joins table controller.
Routes look good and restful.